src/response.rs

```
@@ -124,46 +123,52 @@ impl<'a, 'b> Response<'a, 'b> {
     ///     response.render("examples/assets/template.tpl", &data);
     /// }
     /// ```
-    pub fn render<'c, T: Encodable<Encoder<'c>, Error>>
+    pub fn render<'c, T: Encodable>
         (&mut self, path: &'static str, data: &T) {
             // Fast path doesn't need writer lock
-            match self.templates.read().get(&path) {
-                Some(t) => {
-                    let _ = t.render(self.origin, data);
-                    return
+            match self.templates.read() {
+                Ok(hm) => match hm.get(&path) {
+                    Some(t) => {
+                        let _ = t.render(self.origin, data);
+                        return
+                    },
+                    None => {}
                 },
-                None => {}
+                Err(_) => {}
```

Probably shouldn't eat this error without writing it somewhere